packages/core/src/base.ts

```diff
@@ -58,6 +58,9 @@ const defaultOptions: ILogtailOptions = {
 
   // Function to be used to calculate size of logs in bytes (to evaluate batchSizeLimitKiB)
   calculateLogSizeBytes: calculateJsonLogSizeBytes,
+
+  // Use IPv6 for sending logs to Better Stack. Enable this if you are running on nodejs in an IPv6-only network.
+  useIPv6: false,
```

Comment:
The `useIPv6` is defined in the common `core`, but it's not respected in `browser` and `edge` variants.

We already have package-specific options (see `ILogtailEdgeOptions`), so it would be possible to define it only for the `node` package following the same pattern.

Alternatively, the `useIPv6` can be used in all 3 defined sync functions. WDYT?

Reply:
Ah, sorry, missed the package-specific options; let's just do it for node, the other packages won't have this problem, it's node-only. Will update.